substrafl/evaluation_strategy.py

```diff
@@ -97,6 +97,15 @@ def __init__(
             elif not all(r >= 0 for r in rounds):
                 raise ValueError(f"rounds can be only positive, rounds={rounds} found")
 
+    @property
+    def test_data_nodes_org_ids(self):
+        """Property to get the ids or test data nodes organizations.
+
+        Returns:
+             set: set of organization ids
```

Comment:
Out of curiosity, why don't you annotate the return type? (`-> set`)

Reply:
It was not on purpose, I'm updating with it ! thx :)